test/trains.js

```diff
@@ -1,12 +1,48 @@
 // import chai, { expect } from 'chai'
 // import Trains from '../db/commands/trains'
 const chai = require( 'chai' )
-const expect = require( 'chai' ).expect
-const Trains = require( '../db/commands/trains' )
+const expect = chai.expect
```

Comment:
you could make this one line by either doing:
const chai = require('chai').expect
const {expect} = require('chai')